11/task1.py

```diff
@@ -0,0 +1,29 @@
+"""
+Home Work 11
+Dmytro Verovkin
+robot_dreams
+
+Написати власний декоратор, задачею якого має бути друк назви функції і часу, коли вона була викликана.
+Декоратор має працювати для різних функцій однаково.
+"""
+
+
+from datetime import datetime
+
+def my_decorator(func):
+    def wrap(*args, **kwargs):
+        print(f"'{func.__name__}' called at {datetime.now().strftime('%Y-%m-%d %H:%M:%S')}")
```

Comment:
You can also try to use `datetime.now().isoformat()` which will give you almost the same format